docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,89 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-howto"]
+= Update projects to Quarkus 3 automatically
+:extension-status: preview
+include::_attributes.adoc[]
+:categories: core
+:summary: Update projects from Quarkus 2.13 and later to Quarkus 3.
+
+As a developer, you can update your projects from Quarkus 2.13 and later to Quarkus 3 by running one of the update commands.
+These commands use OpenRewrite recipes to automatically update most of your project's dependencies, source code, and documentation.
+
+//  For existing applications where all extensions are available, we have an OpenRewrite recipe you can try. GIVE LINK
+//
+// Once this has been run, your project should update its dependencies, source code, and documentation to use Quarkus 3.
+//
+// If it does not work for you, we could have missed something, or you are using extensions not yet supporting Quarkus 3.
+//
+// https://github.com/quarkusio/quarkus/issues[Let us know] in either case so we can improve the migration script together.
+
+
+[IMPORTANT]
+====
+After updating, evaluate whether your project requires additional follow-up steps.
+Verify that the application from your updated project works as expected before releasing it.
+====
+
+:extension-status: "experimental"
+include::{includes}/extension-status.adoc[]
+
+== Prerequisites
+
+:prerequisites-time: 30 minutes
+include::{includes}/prerequisites.adoc[]
+
+== Procedure
+
+. If your project uses Hibernate ORM or Hibernate Reactive, see the link:https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration[Hibernate ORM 5 to 6 migration] quick reference.
```

Comment:
Is the goal/action of step 1 to make sure you are using version 6 of Hibernate ORM? If so, should we be direct here about the required action. Also, wondering if this is more of a prereq?

Reply:
I'll ask @yrodiere for clarification about the best placement of this information.